controller/config/samples/dagTasks/kontroler_v1alpha1_dagtask.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: kontroler.greedykomodo/v1alpha1
+kind: DagTask
+metadata:
+  labels:
+    app.kubernetes.io/name: dagtask
+    app.kubernetes.io/instance: dagtask-sample
+  name: dagtask-sample
+spec:
+  command: ["sh","-c"]
```

yamllint (1.35.1)

[warning] too few spaces after comma

(commas)